spec/features/display_all_peeps_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'Peeps Page' do
+  scenario 'User goes to peeps page and sees all peeps listed' do
```

Comment:
I do not think this scenario is truly tested as it does not check for the page to have peep content, just a title.